test/integration/targets/lookup_mongodb/tasks/test_lookup_mongo.yml

```diff
@@ -0,0 +1,62 @@
+- name: "assert we complain properly if pymongo is not there"
+  debug: msg="{{ lookup('mongodb', mongodb_parameters ) }}"
+  register: result
+  ignore_errors: yes
+
+- name: "assert we complain properly if pymongo is not there (assert part)"
+  assert:
+    that:
+    - result is failed
+    - result is not changed
+    - "'pymongo is required' in result.msg"
+
+- name: install pymongo
+  pip:
+    name: "pymongo"
+
+- name: "assert we complain properly if we can't connect to mongo"
+  debug: msg="{{ lookup('mongodb', mongodb_parameters ) }}"
+  register: result
+  ignore_errors: yes
+
+- name: Assert we fail with the right message if we can't connect to mongodb
+  assert:
+    that:
+    - result is failed
+    - result is not changed
+    - "'unable to connect to database' in result.msg"
+
+- name: "Copying our custom /etc/mongodb.conf which does not use as much disk space and makes mongo start quickly"
+  copy:
+    src: mongodb.conf
+    dest: /etc/mongodb.conf
+
+- name: Installing mongodb
+  # somehow the apt module messes up all tests...
+  command: "apt-get install -o Dpkg::Options::=\"--force-confold\" --force-yes -y mongodb"
```

Comment:
What issue(s) did you encounter when trying to use the `apt` module?

Reply:
@marcosdiez Could we please get an answer to this?

Reply:
That's a hard one. I don't remember. I wrote this code in October. If this is blocking the PR from being merged, I can try to reproduce the issue. In my humble opinion this is secondary, since 
- this is only a test and not the main code itself
- yes, it could be better but it actually works

Reply:
Agreed, I don't think this blocks